virttest/graphical_console.py

```diff
@@ -0,0 +1,97 @@
+"""
+Graphical Console
+"""
+
+import time
+
+
+def uniform_linear(duration, rate):
+
+    def motion(src, dst):
+        pass
+
+    return motion
+
+
+class DummyConsole(object):
+
+    """
+    Dummy console
+    """
+
+    KEY_DOWN = True
+    KEY_UP = False
+
+    BTN_DOWN = True
+    BTN_UP = False
+
+    SCROLL_FORWARD = 1
+    SCROLL_BACKWARD = -1
+
+    def __init__(self, vm, logfile=None):
+        self._vm = vm
```

Comment:
Seems not `vm` not really referenced in this class, but only at QMPInputConsole, I suppose it would be better to assign it there, and also no where reference `logfile`, it's meaningless to pass it.

Reply:
updated, thanks~